src/components/importantDetails/ImportantDetails.vue

```diff
@@ -18,39 +18,39 @@
         <tr>
           <th scope="row">Avarage seconds per block</th>
           <td>~ {{ sharedState.rskConfig.secondsPerBlock }} seconds</td>
-          <td>~ {{ sharedState.ethConfig.secondsPerBlock }} seconds</td>
+          <td>~ {{ sharedState.sideConfig.secondsPerBlock }} seconds</td>
         </tr>
         <tr>
           <th scope="row">Confirmations needed for small amounts</th>
           <td>
             {{ rskConfirmations?.smallAmount }} blocks (~ {{ rskConfirmations?.smallAmountTime }}) +
-            2 minutes voting
+            {{ votingTime }}
```

Comment:
maybe add an indication of the time unit? like `{{ votingTime }} minutes`

Reply:
![image](https://user-images.githubusercontent.com/17556614/133784765-a0994fbf-8c15-4fb7-ac28-d085dfb01431.png)
It is done in another place